src/doom/d_main.c

```diff
@@ -1569,6 +1571,10 @@ void D_DoomMain (void)
 #endif
 
     DEH_printf("W_Init: Init WADfiles.\n");
+    
+    gpio_init(PICO_DEFAULT_LED_PIN);
```

Comment:
this causes the LED on the Pico (and Lilygo TT-GO and anything else that uses the default LED pin) to immediately turn on and stay on if something goes wrong loading the WAD file. If the LED just barely briefly flashes when you plug in the pico you know the wad load was successful (or at least, didn't error)